File: 01 -Nivelamento/unidade00o_processoAEDIIExercicios/08 - Dona Mônica em C-C++/DonaM.c
#include <stdio.h>

int main() {
    int donaM = -1, filhoVelho = 0, filhoUm = 0, filhoDois = 0;
    while (donaM != 0) {
        scanf("%d", &donaM);
        scanf("%d", &filhoUm);
        scanf("%d", &filhoDois);

        filhoVelho = donaM - (filhoUm + filhoDois);
        if (donaM != 0) {
            printf("%d\n", filhoVelho);
        }
    }

    return 0;
}

File: 01 - Nivelamento/Exercicios/unidade00o_processoAEDIIExercicios/07 - Dona Mônica em Java/DonaM.java
public class DonaM {
    public static void main(String[] args) {
        int donaM = -1, filhoVelho = 0, filhoUm = 0, filhoDois = 0;
        while (donaM != 0) {
            donaM = MyIO.readInt();
            filhoUm = MyIO.readInt();
            filhoDois = MyIO.readInt();
            

            filhoVelho = donaM - (filhoUm + filhoDois);
            if (donaM != 0) {
                MyIO.println(filhoVelho);
            }
        }
    }
}
